Gemfile

```diff
@@ -25,6 +25,7 @@ gem 'kaminari'
 gem 'soft_deletion', '~> 0.4'
 gem 'dalli', '~> 2.7.0'
 gem 'active_model_serializers', '~> 0.8.0'
+gem 'nokogiri', '1.6.7.1'
```

Comment:
Yup, agree. Specific version is only required if the gem is already being locked with the old version... In this case, the Gemfile.lock + the .gem are the only required changes. Will update the PR. Thanks for the feedback!